examples/arduino-mega2560/src/bin/mega2560-rgb-led.rs

```diff
@@ -0,0 +1,67 @@
+#![no_std]
```

Comment:
If possible, would you mind moving this example over to Arduino Uno?  That board is sort of our reference platforms and I'd prefer to have most of the examples there.  This will help users to find them all in one place.

Reply:
I don't have an arduino uno to test it on :(

Reply:
To be honest, then I'd suggest to keep the examples in the Arduino Mega 2560 folder.  I'd rather have them where they were tested and confirmed to work then to move them over.  In the future, someone else can always port them to Uno if the need arises.

Reply:
Okay sounds good, in the future I may get the uno board and do it myself